packages/gatsby/src/redux/persist.ts

```diff
@@ -58,7 +59,9 @@ export function readFromCache(
 
     return v8.deserialize(
       readFileSync(
-        reduxWorkerSlicesPrefix(cacheFolder) + createContentDigest(slices)
+        reduxWorkerSlicesPrefix(cacheFolder) +
+          `${optionalPrefix}_` +
+          createContentDigest(slices)
```

Comment:
So for the main process filename will be like: `_abc` and for workers: `1_abc`?

Reply:
Yes, correct. I didn't use `main` as this might be used in other contexts aside from "main", "worker" etc. and also didn't want to use `0` in case the IDs are zero-indexed

Reply:
I mean in the calling function, not the method signature?

Reply:
Anyways, probably fine like that. Just aesthetically weird when files are like that:

`_abc`
`1_abc`
`2_abc`

Looks like an error :) But I can live with that.

Reply:
Ah, I see. I didn't see any value in this as we can distinguish between `__hash` and `_1_hash` and the `.cache/worker` directory doesn't contain anything else.